### PR TITLE
More content updates

### PR DESCRIPTION
**Pull request by pcraig3**
A whole whack of small changes.

- removed 'Optional' hints
- removed `userNeeds` question (not in the prototype)
- locations questions had mixed up `question` and `question_advice` keys
- cultural fit criteria questions were using the wrong `question`

Was also using sublime to edit stuff, and I've configured it to delete trailing whitespace, so that's why the diff is so hard to read.

### CODE REVIEW

File: frameworks/digital-outcomes-and-specialists/questions/briefs/culturalFitCriteriaOutcomes.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+question: Cultural fit evaluation criteria
 question_advice: |
   Cultural fit is how well you and the individual or team will work together. If the supplier’s values and behaviour
   are similar to your organisation’s, they should be a good cultural fit.
-  
-  If you want to add ‘points’ to the individual cultural fit criteria, you must include them here, 
-  eg ‘transparent and collaborative when making decisions - 15 points’
-  
+
+  You must tell suppliers if you want to score individual criteria.
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/culturalFitCriteriaOutcomes.yml]: I think we want a line break before the link here

File: frameworks/digital-outcomes-and-specialists/questions/briefs/culturalFitCriteriaSpecialists.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+question: Cultural fit evaluation criteria
 question_advice: |
   Cultural fit is how well you and the individual or team will work together. If the supplier’s values and behaviour
   are similar to your organisation’s, they should be a good cultural fit.
-  
-  If you want to add ‘points’ to the individual cultural fit criteria, you must include them here, 
-  eg ‘transparent and collaborative when making decisions - 15 points’
-  
+
+  You must tell suppliers if you want to score individual criteria.
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/culturalFitCriteriaSpecialists.yml]: And here

File: frameworks/digital-outcomes-and-specialists/questions/briefs/locationOutcomes.yml [hunk 3]
```diff
@@ -1,10 +1,12 @@
-name: Where you want the supplier to work
+name: Region
+question: Where you want the supplier to work
 id: location
-question: |
+question_advice: |
   This should be the primary location where they’ll be working.
-  
-  Suppliers have said where they’ll provide services.
-hint: "[View region map (PDF, 262KB)](https://geoportal.statistics.gov.uk/Docs/Maps/Regions_(Eng)_2011_map.pdf)"
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/locationOutcomes.yml]: Wot no PDF?

**Comment by TheDoubleK** [comment]
I never knew you hate whitespace so much.  Or is that PyCharm?

**Comment by pcraig3** [comment]
Sublime text.  When it saves files it (a) cuts out trailing whitespace and (b) adds a newline if there wasn't one.

**Comment by allait** [comment]
👍

**Pull request merged**